### PR TITLE
Shorten notebook titles

### PR DESCRIPTION
**Pull request by tomflexcompute**
Shortened notebook titles as discussed.

### CODE REVIEW

File: docs/source/notebooks/BoundaryConditions.ipynb
Comment (tylerflex) [inline docs/source/notebooks/BoundaryConditions.ipynb:1]: Setting up boundary conditions

File: docs/source/notebooks/CustomFieldSource.ipynb
Comment (tylerflex) [inline docs/source/notebooks/CustomFieldSource.ipynb:1]: Defining complex, spatially-varying sources

File: docs/source/notebooks/CustomMediumTutorial.ipynb
Comment (tylerflex) [inline docs/source/notebooks/CustomMediumTutorial.ipynb:1]: Defining ...

File: docs/source/notebooks/Dispersion.ipynb
Comment (tylerflex) [inline docs/source/notebooks/Dispersion.ipynb:1]: Modeling ...

File: docs/source/notebooks/Fitting.ipynb
Comment (tylerflex) [inline docs/source/notebooks/Fitting.ipynb:1]: Fitting ...

File: docs/source/notebooks/FieldProjections.ipynb
Comment (tylerflex) [inline docs/source/notebooks/FieldProjections.ipynb:1]: Performing ...

File: docs/source/notebooks/FullyAnisotropic.ipynb
Comment (tylerflex) [inline docs/source/notebooks/FullyAnisotropic.ipynb:1]: Defining ...

File: docs/source/notebooks/GDS_import.ipynb
Comment (tylerflex) [inline docs/source/notebooks/GDS_import.ipynb:1]: Defining

**Review by tylerflex — CHANGES_REQUESTED** [review]
Thanks @tomflexcompute just one general comment, any title that starts with a verb (most of them) for example "Define", "Model", "Perform", I think we need to make in the tense like "defining", "modeling", "perform". It will sound more natural I think. Otherwise looks good! I looked through all but didn't note them after a while since most needed this change.

File: docs/source/notebooks/WaveguideCrossing.ipynb
Comment (tylerflex) [inline docs/source/notebooks/WaveguideCrossing.ipynb:1]: fix the `oon`

**Review by tylerflex — APPROVED** [review]
Just one typo in WaveguideCrossing otherwise looks good thanks!

**Comment by tomflexcompute** [comment]
Thanks @tylerflex !

**Pull request merged**